images/plbase/python-requirements.txt

```diff
@@ -1,41 +1,41 @@
 ansi2html==1.9.2
 chevron==0.14.0
-coloraide==3.3.1
-Faker==28.1.0
+coloraide==4.0
```

Comment:
The breaking changes were pretty minimal for this, I don't think anyone is using the color science features that got messed with.

Reply:
Man, Faker really loves their major versions...